src/test/java/de/undercouch/actson/PrettyPrinter.java

```diff
@@ -23,12 +23,11 @@
 
 package de.undercouch.actson;
 
-import java.nio.charset.StandardCharsets;
```

Comment:
My motivation was that since I need correct serialization of escaped chars back to JSON I'd need to modify this printer accordingly which seemed like unnecessary work since it's just demo and test code - not the core functionality of Actson. Also I think that it is a good idea to not have duplicating functionality in demo and test code. 

That said, this version is required for tests to run correctly. If you want to still have have demo code without extra dependencies then probably I should keep older version as a separate class (maybe with caveat that it does not handle escapes).